source/nodejs/aclint/src/index.ts

```diff
@@ -1,256 +0,0 @@
-#!/usr/bin/env node
-
-import * as yargs from 'yargs';
```

Comment:
Don't bring clang-format, js-ci, removing aclint, other changes into this PR

Reply:
reverted thanks!